app/react-native/package.json

```diff
@@ -33,8 +33,8 @@
     "rn-host-detect": "^1.1.5"
   },
   "devDependencies": {
-    "@types/react-native": "^0.57.57",
-    "react-native": "^0.57.8"
+    "@types/react-native": "^0.57.64",
+    "react-native": "^0.59.9"
```

Comment:
our peer dependency still says `"react-native": ">=0.57.0"` but it looks like we broke backward compatibility with older version of RN. We may need a deeper look to understand why that happened to see if it can be fixed. With our next major release, we may want to document what version of RN/expo we're going to support and for how long. We should be able to support 0.59 for longer than 0.57 since that's the first version with react hooks support.

Reply:
yeap It seems like more investigation is needed. I didn't get much from seeing why the deploy/build failed. For now, I don't know what to do next... If you have a guess, can you tell me where to look?

Reply:
There were few lines of changes yet I didn't see this one. :/
What exactly is the reason to currently lock it to .57 ? What breaking change did we introduce?
And do we need to lock it to .59? If there is need we can do it for next major version I guess.
But I don't think that should be the reason why expo build is failing. I could try to take a look into it too.

Reply:
the main one is some of the storybook core code started using hooks which weren't supported before 59

Reply:
Do you know which part? Maybe we can do something about it? Because if we can't change it we have to lock the version and notify people about it.

Reply:
I'm not really sure, this is mostly a guess 😅

Reply:
Sorry guys. CI failed because of source-loader problem. After I merged latest `next` branch, there was no test fail anymore.